src/web/client/common/constants.ts

```diff
@@ -12,7 +12,8 @@ export const NO_CONTENT = ' ';
 export const EMPTY_FILE_NAME = 'defaultfilename';
 export const CHARSET = 'utf-8';
 export const PROVIDER_ID = "microsoft";
-export const SCOPE_OPTION = "//.default";
+export const SCOPE_OPTION_DEFAULT = "//.default";
```

Comment:
I think this value should be ".default". Can you please check and confirm.
For reference: https://learn.microsoft.com/en-us/power-apps/developer/data-platform/authenticate-oauth#simple-example

I know the value that you have used is same as it was earlier.

Reply:
//.default and /.default work  - .default and ///.default dont